cmd/application/conns.go

```diff
@@ -254,6 +257,9 @@ readLoop:
 			} else if errors.Is(err, errConnTimeout) {
 				logger.Errorf("error occurred discarding data (read %d B): timeout\n", received.Len())
 				cm.discardToTimeout(asn, cc, isIPv4)
+			} else if errors.Is(err, errConnClosed) {
+				logger.Errorf("error occurred discarding data (read %d B): closed\n", received.Len())
```

Comment:
I am not sure that it is necessary to log this text since this is the default expected outcome for things that end up in discard. Essentially this only tells us the transition and the number of bytes read, but takes a whole log line that we are not currently parsing. :shrug: